job-service-db/src/main/resources/changelog.xml

```diff
@@ -152,4 +152,11 @@
                          schemaName="public">
         </createProcedure>
     </changeSet>
-</databaseChangeLog>
\ No newline at end of file
+
+    <changeSet id="create_procedure_getIdentityBasedTableName" runOnChange="true" author="Anthony McGreevy" >
```

Comment:
Why are you modifying this one instead of `procedures/changelog.xml`?

Reply:
Hadn't seen the changelog.xml under the procedures folder.